### PR TITLE
Make database seed work if Unsplash down

### PR DESCRIPTION
**Pull request by krystof-k**
Well, Unsplash is [down](https://source.unsplash.com) so just a little rescue in the seeds script…

### CODE REVIEW

File: spec/dummy/db/seeds.rb [hunk 1]
```diff
@@ -165,7 +165,15 @@
...
-  post.cover_photo.attach(io: URI.open("https://source.unsplash.com/random/#{[1000, 1100, 1200, 1300].sample}x#{[1000, 1100, 1200, 1300].sample}/?all"), filename: "cover.jpg")
+  begin
+    width = [1000, 1100, 1200, 1300].sample
+    height = [1000, 1100, 1200, 1300].sample
+    url = "https://source.unsplash.com/random/#{width}x#{height}/?all"
+    io = URI.open(url) # standard:disable Security/Open
+  rescue
+    puts "Failed to fetch cover photo from Unsplash"
+  end
+  post.cover_photo.attach(io: io, filename: "cover.jpg") if io
```
Comment (adrianthedev) [inline spec/dummy/db/seeds.rb:176]:
```suggestion
  begin
    width = [1000, 1100, 1200, 1300].sample
    height = [1000, 1100, 1200, 1300].sample
    url = "https://source.unsplash.com/random/#{width}x#{height}/?all"
    io = URI.open(url) # standard:disable Security/Open

    post.cover_photo.attach(io: io, filename: "cover.jpg")
  rescue
    puts "Failed to fetch cover photo from Unsplash"
  end
```

Comment (adrianthedev) [reply spec/dummy/db/seeds.rb:176]: Could this work as well?

Comment (krystof-k) [reply spec/dummy/db/seeds.rb:176]: Yeah but in case just the attach fails the error message isn't true :)

Comment (adrianthedev) [reply spec/dummy/db/seeds.rb:176]: Got it! makes sense!
Ok. let's go with the original version.

**Pull request merged**